documentation/modules/rn-2.2.adoc

```diff
@@ -5,25 +5,37 @@
 [id="rn-22_{context}"]
 = {project-full} 2.2
 
-You can migrate virtual machines (VMs) from VMware vSphere or {rhv-full} to {virt} with {the-lc} {project-first}.
+This release supports migration of virtual machines (VMs) from VMware vSphere or {rhv-full} to {virt} with {the-lc} {project-first}.
```

Comment:
Please revert this sentence. We do not use the "<inanimate object> does <task>".